fladgejt/webui/predmety.py

```diff
@@ -17,12 +18,11 @@ def _open_register_predmetov(self):
     def get_informacny_list(self, kod_predmetu, akademicky_rok):
         app = self._open_register_predmetov()
 
-        index = find_option(
-            app.d.akRokComboBox.options, title=akademicky_rok)
-        app.d.akRokComboBox.select(index)
+        stredisko, skratka, _ = decode_key(kod_predmetu)[0].split('/')
```

Comment:
> Pozor - myslím, že skratka môže obsahovať '/'. Preto tam pôvodne bol ten `'/'.join(...)`.

Hmm... Tak taky predmet som doteraz nevidel, samozrejme zmenim to :-) 

> Mimochodom: V skutočnosti sú aj predmety, ktoré sa líšia iba v tom poslednom čísle. :-( 

Ano, vsimol som si, no po asi 10min hladania som nasiel mozno 2 predmety ktore sa lisili iba v poslednom cisle. (Vo vacsine pripadov stredisko + skratka postacuje). Chcel som aj napisat do PR, ze neviem ako filtrovat to cislo, ale akosi som na to zabudol.